### ISSUE TITLE
handle ``in`` and ``not in`` seperately

### ISSUE THREAD

**Issue by loechel** [zopefoundation/RestrictedPython#113]
In test we should check following cases seperately:

```python
a in my_object
a in my_list
a in my_dict
a in my_set
```

at they might use `__contains__` or something else, depending on the protocol.

**Comment by loechel** [comment]
Partial check for that issue in #132 

The Check for:
```python
a in my_object
```

should be handled separatly.

**Comment by loechel** [comment]
This issue is partial solved in #132 and the rest is moved to a new issue #139
